web/src/lamda-services/send-email.ts

```diff
@@ -0,0 +1,26 @@
+import nodemailer from 'nodemailer';
+
+export const sendEmail = async ({ name, email, content }: { name: string; email: string; content: string }) => {
+  let transporter = nodemailer.createTransport({
+    service: 'gmail',
+    auth: {
+      user: process.env.GMAIL_USER,
+      pass: process.env.GMAIL_PASS,
+    },
+  });
+
+  let mailOptions = {
+    from: email,
+    to: process.env.GMAIL_USER,
+    subject: 'Email from ' + name,
+    text: `El usuario ${name} envio el siguiente mensaje: ${content}. Responder a ${email}`,
+  };
+
+  transporter.sendMail(mailOptions, function (error, info) {
```

Comment:
this uses a callback, wrap everything into a promise, resolve inside the callback and await that promise so you can still use the async/await syntax. as the code is right now, the api call can return back before the callback gets executed)